### ISSUE TITLE
Advance Import not working

### ISSUE THREAD

**Issue by CyrusiaYini** [lvandernoll/pokecardmaker.net#384]
**Description**
i have the json thing on the clipboard, but nothing happens other than it saying it needs access to my clipboard.
**Screenshots**
...

**Comment by lvandernoll** [comment]
Did you give it access to your clipboard? If so, could you paste the JSON here so I can check it out?

**Comment by JoudKK13** [comment]
I have the same problem too and i don't know how to give it access

**Comment by vandeponcho** [comment]
Some browsers disable read-access to the clipboard by default (e.g. Firefox). Thus, you can export the card, but not import it. There are some workarounds for this, but they are generally not recommended due to security risks (e.g. toggling dom.events.asyncClipboard.readText in Firefox). Try importing in another browser; I had to use Safari instead of Firefox to make it work.

A new button/option to browse a text file on your computer with the card's code, just like with the images and icons, would be awesome. And/or a text box to paste the text using a classic keyboard shortcut (e.g. ctrl+V).

**Comment by lvandernoll** [comment]
@vandeponcho Yeah that could indeed be the problem. I'm not going to spend a lot of time on improving import/export because the upcoming card gallery ([https://test.pokecardmaker.net/](https://test.pokecardmaker.net/)) would make the feature redundant

**Comment by nonchris** [comment]
Maybe just allow to enter a file (with a file dialogue) instead of using the clipboard?
That would fix those security problems ^^
